readme.md

```diff
@@ -29,5 +30,23 @@ If permission is required then
 
 ```chmod +x scripts/run.sh```
 
+or 
+
+```go run main.go```
+
+http://localhost:8000/parse
+
+Copy the url and paste it into Postman, while sending url select the POST method and then select the ```Body >> raw```
```

Comment:
Why copy-paste? This should be already there.
Just point to the API definition file (the postman exported json file).